ops/manifests/manifest.py

```diff
@@ -343,5 +343,22 @@ def delete_resources(
                     log.exception(f"{log_msg}")
                     raise ManifestClientError(log_msg) from ex
 
+    def _delete(
+        self, obj: HashableResource, namespace: Optional[str], ignore_labels: bool
+    ):
+        if ignore_labels:
+            self.client.delete(type(obj.resource), obj.name, namespace=namespace)
```

Comment:
This is the old behavior, to just delete a resource regardless of who owns it.  Callers to `delete_resources` will need to opt in for this dangerous behavior